.github/workflows/integration-tests.yml

```diff
@@ -22,15 +22,21 @@ jobs:
   integration-tests:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        browser: ["chrome", "firefox"]
```

Comment:
Will this double integration running time? I've had integration tests take ~15 minutes to run before, so doubling that would be impactful.  

I'm open to setting this here for now and seeing how it goes. Another option would be to run firefox on merging to main rather than on every pull request, since I'm hoping it's unlikely chrome will succeed and firefox will fail.

Reply:
Oh on this PR it looks like they may be running in parallel. That would be great.

Reply:
no the matrix setup makes the tests run in parallel so I don't expect an increase.
On a test run it took longer on firefox but I am not sure why, https://github.com/globaldothealth/list/actions/runs/212060717

let me see what the results are for the current run...

Reply:
hmm they actually took exactly the same time on that last run so I think we're good
https://github.com/globaldothealth/list/actions/runs/212095963